ntoskrnl/ke/i386/kiinit.c

```diff
@@ -824,6 +825,16 @@ KiSystemStartup(IN PLOADER_PARAMETER_BLOCK LoaderBlock)
     __writefsdword(KPCR_SET_MEMBER_COPY, 1 << Cpu);
     __writefsdword(KPCR_PRCB_SET_MEMBER, 1 << Cpu);
 
+    //TODO: We don't setup IPIs yet so freeze other processors here.
+    if (Cpu)
+    {
+        for(;;)
```

Comment:
Put a `LoaderBlock->Prcb = NULL; KeMemoryBarrier();` here to allow P0 to continue.